projects/demo/src/modules/components/input-chip/examples/3/index.html

```diff
@@ -0,0 +1,14 @@
+<tui-textfield multi>
+    <input
+        placeholder="Enter"
+        tuiInputChip
+        [unique]="false"
+        [(ngModel)]="value"
+    />
+
+    <tui-input-chip-item
```

Comment:
This name looks weirdly long 🤔 Just a thought

Reply:
wdyt about `tui-chip-item` ?

Reply:
decided to name it the `tui-input-chip`